### PR TITLE
Charts review

### PR DESCRIPTION
**Pull request by kghamilton89**
The second half of the earlier (now closed) PR. This PR includes only grammatical changes, included the issues you raised in the now-closed PR.

### CODE REVIEW

**Comment by kghamilton89** [comment]
For some reason "format-repair" commit was included in this PR, but I've reverted the changes so they remain split.

File: syntax/format-settings.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Value | Syntax | Description
 `binary` | `format = binary('kilo', 3) + 'B'` | Formatting values with size units. Used for bytes.<br>For example, `1024`.
 `decimal`| `format = numeric(1)`<br>`format = numeric`| Number formatting: `numeric` is an `alias` for `decimal`.<br>Specify the optional round value parameter in round brackets to define how many spaces the decimal is rounded.<br>Example: `format = numeric(1)` rounds to the first decimal. Thus, `4.29` becomes `4.3`.
 `fixed`| `format = fixed(3)` | Format numbers with the specified number of digits.<br>Example: `format = fixed(1)` rounds `1225` to `1000`.
-`currency` | `format = '$' + currency('million')`| Currency formatting. Any symbol is supported.
+`currency` | `format = '$' + currency('million')`| Currency formatting.<br>Any currency symbol is supported.<br>Indicate abbreviated numbers by including the shortened denomination as an argument in the `currency` expression.<br>For example: `format = '$' + currency('million')` renders the raw value `6.3` as `$6.3M`.
```
Comment (rodionos) [inline syntax/format-settings.md]: > `format = '$' + currency('million')` renders the raw value `6.3` as `$6.3M`.
It works somewhat differently: https://apps.axibase.com/chartlab/03f7f76f

File: syntax/selecting-series.md [hunk 2]
```diff
@@ -290,4 +290,4 @@ display = value == top(1) || value == bottom(1)
 
 [![](./images/button.png)](https://apps.axibase.com/chartlab/cdfb34c5/13/)
 
-In addition, the `limit = int` setting reduces the number of samples displayed for each series. It makes queries execute faster when loading data for high-frequency series from the server, in particular during design and validation stages.
\ No newline at end of file
+In addition, the `limit = int` setting reduces the number of samples displayed for each series. The `limit = int` setting aids in quicker query execution when loading data for high-frequency series from the server, in particular during the design and validation stages.
```
Comment (rodionos) [inline syntax/selecting-series.md]: Is there a semantic difference between quicker and faster? I've only seen ...faster execution, etc.?

File: syntax/thresholds.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 # Thresholds
 
-Use client-side threshold aggregators to compute the amount of time or number of occurrences a metric violates a user-defined threshold.
-
-Specify lower or upper thresholds or combine the two to trigger a violation when a series value is exceeds the specified range.
+Use client-side threshold aggregators to compute the amount of time or number of occurrences a metric violates a user-defined threshold. The threshold can be defined as an upper boundary, a lower boundary, or both.
```
Comment (rodionos) [inline syntax/thresholds.md]: Why only client-side? Our api supports these aggregators as well.

File: syntax/format-settings.md (hunk 1 above)
Comment (kghamilton89) [reply syntax/format-settings.md]: Added a note that values are rounded as well.

File: syntax/selecting-series.md (hunk 2 above)
Comment (kghamilton89) [reply syntax/selecting-series.md]: No semantic difference, I'll revert it to faster.

File: syntax/thresholds.md (hunk 3 above)
Comment (kghamilton89) [reply syntax/thresholds.md]: Removed.

**Pull request merged**